asmcomp/cmm.mli

```diff
@@ -134,10 +134,10 @@ and expression =
   | Clet of Ident.t * expression * expression
   | Cassign of Ident.t * expression
   | Ctuple of expression list
-  | Cop of operation * expression list
+  | Cop of operation * expression list * Debuginfo.t
   | Csequence of expression * expression
   | Cifthenelse of expression * expression * expression
-  | Cswitch of expression * int array * expression array
+  | Cswitch of expression * int array * expression array * Debuginfo.t
```

Comment:
I probably completely miss the context of the PR, but from a user perspective, being able to put a breakpoint before an assignment, show the current value of the reference, step through the assignment, and show the new value seem like a basic use of a debugger; and this should not be broken if the reference is turned into a local variable by the compiler.  But perhaps this can be achieved already without keeping any more information.